input/kafkamdm/kafkamdm.go

```diff
@@ -366,6 +327,17 @@ func (k *KafkaMdm) handleMsg(data []byte, partition int32) {
 		return
 	}
 
+	isControlMsg := msg.IsIndexControlMsg(data)
+	if isControlMsg {
+		cm, err := msg.ReadIndexControlMsg(data)
+		if err != nil {
+			log.Errorf("kafkamdm: decode error, skipping control message. %s", err)
```

Comment:
just like the case below where we can't decode a MetricData, we should increment a similar error metric , e.g. `controlMsgDecodeErr.Inc()`